guides/theming-your-components.md

```diff
@@ -3,31 +3,30 @@ In order to style your own components with Angular Material's tooling, the compo
 be defined with Sass.
 
 #### 1. Define all color and typography styles in a "theme file" for the component
-First, create a Sass mixin that accepts an Angular Material theme and outputs the color-specific
-styles for the component. An Angular Material theme definition is a Sass map.
+First, create a Sass mixin that accepts an Angular Material color configuration and
+outputs the color-specific styles for the component. A color definition is a Sass map.
```

Comment:
The top sentence calls it a "color configuration" and the bottom one calls it "color definition" which is inconsistent.

Reply:
Fixed in the other PR that changed this